src/freetype/ft_render.c

```diff
@@ -421,12 +422,20 @@ SDL_Surface *_PGFT_Render_NewSurface(FreeTypeInstance *ft,
     FT_UInt32 bmask = 0x00ff0000;
     FT_UInt32 amask = 0xff000000;
 #endif
+#else /* IS_SDLv2 */
+    FT_UInt32 rmask = 0;
```

Comment:
I don't understand why for SDL2 these are all 0?

Reply:
Maybe freetype is using pallette rendering in sdl2? 
A pixel format has either a palette or masks. If a palette is used Rmask, Gmask, Bmask, and Amask will be 0.